### PR TITLE
Init modules mechanism

### PR DESCRIPTION
**Pull request by DamnClin**
Related to https://github.com/jhipster/jhipster-lite/issues/1768

### CODE REVIEW

**Comment by DamnClin** [comment]
> can you add in description the link to the ticket plz? Is it #1666 or #1768 or both ? :D

None :D. @hdurix ask for a smaller PR so it's just a part of https://github.com/jhipster/jhipster-lite/issues/1768

File: src/main/java/tech/jhipster/lite/common/domain/JhipsterCollections.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package tech.jhipster.lite.common.domain;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class JhipsterCollections {
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/common/domain/JhipsterCollections.java:6]: JhipsterCollections -> JHipsterCollections

File: src/test/java/tech/jhipster/lite/common/domain/JhipsterCollectionsTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import tech.jhipster.lite.UnitTest;
+
+@UnitTest
+class JhipsterCollectionsTest {
```
Comment (pascalgrimaud) [inline src/test/java/tech/jhipster/lite/common/domain/JhipsterCollectionsTest.java:13]: JhipsterCollectionsTest -> JHipsterCollectionsTest

File: src/test/java/tech/jhipster/lite/generator/module/infrastructure/secondary/JHipsterModulesAssertions.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  public static class ModuleFileAsserter {
+
+    private final ModuleAsserter moduleAsserter;
+    private final String file;
+
+    private ModuleFileAsserter(ModuleAsserter moduleAsserter, String file) {
+      this.moduleAsserter = moduleAsserter;
+      this.file = file;
+      assertThat("file").as("Can't check file without file path").isNotBlank();
```
Comment (pascalgrimaud) [inline src/test/java/tech/jhipster/lite/generator/module/infrastructure/secondary/JHipsterModulesAssertions.java:74]: maybe instead ?

```suggestion
      assertThat(file).as("Can't check file without file path").isNotBlank();
```

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**